rules/macos/persistence_screensaver_engine_unexpected_child_process.toml

```diff
@@ -0,0 +1,49 @@
+[metadata]
+creation_date = "2021/10/05"
+maturity = "production"
+updated_date = "2021/10/05"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies when a child process is spawned by the screensaver engine process, which is consistent with an attacker's
+malicious payload being executed after the screensaver activated on the endpoint. An adversary can maintain persistence
+on a macOS endpoint by creating a malicious screensaver (.saver) file and configuring the screensaver plist file to
+execute code each time the screensaver is activated.
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Unexpected Child Process of Screensaver Engine"
+note = """## Triage and analysis
+
+- Analyze the descendant processes of the ScreenSaverEngine process for malicious code and suspicious behavior such
+as downloading a payload from a server
+- Review the installed and activated screensaver on the host. Triage the screensaver (.saver) file that was triggered to
+identify whether the file is malicious or not.
+"""
+references = [
+    "https://posts.specterops.io/saving-your-access-d562bf5bf90b",
+    "https://github.com/D00MFist/PersistentJXA",
+]
+risk_score = 47
+rule_id = "48d7f54d-c29e-4430-93a9-9db6b5892270"
+severity = "medium"
+tags = ["Elastic", "Host", "macOS", "Threat Detection", "Persistence"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where event.type == "start" and process.parent.name == "ScreenSaverEngine"
+'''
+
+
+[[rule.threat]]
```

Comment:
maybe you can add https://attack.mitre.org/techniques/T1546/ (for both rules)

Reply:
Good idea. I'll add T1546 to both rules. I thought about adding T1546.002, but that's specific to Windows.